apps/ledger-live-desktop/tests/models/Transaction.ts

```diff
@@ -1,10 +1,25 @@
+import { Fee } from "tests/enum/Fee";
 import { Account } from "../enum/Account";
+import { Token } from "../enum/Token";
 
-export class Transaction {
+export abstract class Transaction {
   constructor(
     public readonly accountToDebit: Account,
-    public readonly recipient: string,
+    public readonly accountToCredit: Account,
     public readonly amount: string,
-    public readonly speed: string,
+    public readonly speed: Fee,
   ) {}
 }
+export class BasicTransaction extends Transaction {}
+export class TokenTransaction extends Transaction {
+  constructor(
+    accountToDebit: Account,
+    public readonly accountToCredit1: Account,
```

Comment:
why are there 2 accounts to credit?

Reply:
Because it needs to handle transactions involving two different credit accounts (ETH to BTC - error expected - then ETH to ETH) 
https://ledgerhq.atlassian.net/browse/B2CQA-1904